README.md

```diff
@@ -76,3 +76,56 @@ Make sure you read the `README.md` file in the `reporting` directory and install
 ## Implementing the benchmark for a new tool
 
 To implement a tool, you need to create a new directory in the solutions directory and give it a suitable name.
+
+## Running the benchmark with Docker
+
+Instructions for running the benchmark starting with a fresh Ubuntu 20.04 VM.
+
+Create a file system on `$DEVICE` and mount it to `MOUNT_POINT`.
+
+```bash
+sudo mkfs.ext4 $DEVICE
+sudo mount $DEVICE $MOUNT_POINT
+```
+
+Install docker with `apt`:
```

Comment:
Rather than snaps, it may be better to use the official Docker Debian repositories:

https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository

Users will also get updates more frequently through that repo.